bedrock-entry.sh

```diff
@@ -27,9 +27,12 @@ case ${VERSION^^} in
   1.14|PREVIOUS)
     VERSION=1.14.60.5
     ;;
-  1.16)
+  1.16.1)
     VERSION=1.16.1.02
     ;;
+  1.16.20)
```

Comment:
I think if you change this to simply "1.16" then that'll be a decent compromise for the different clients out there.